src/Tribe/Editor.php

```diff
@@ -211,4 +211,17 @@ public function is_classic_editor() {
 
 		return $is_classic_editor_request || $disabled_by_plugin || $disabled_by_filter;
 	}
+
+	/**
+	 * Returns whether Blocks are being used for Events
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function is_events_using_blocks() {
+		$is_blocks_active_for_events = tribe_is_truthy( tribe_get_option( 'toggle_blocks_editor', false ) );
```

Comment:
Pre-computing this here is not efficient.
Since the condition you're running after is an `&&` the second part will only be evaluated if the first is true. Pulling options involves some filters and, possibly, a db query.
I would do this on the fly.